examples/extracellular/extrafeats_example.ipynb

```diff
@@ -0,0 +1,628 @@
+       " 'peak_trough_ratio',\n",
+       " 'repolarization_slope',\n",
+       " 'recovery_slope',\n",
+       " 'neg_peak_relative',\n",
+       " 'pos_peak_relative',\n",
+       " 'neg_peak_diff',\n",
+       " 'pos_peak_diff',\n",
+       " 'neg_image',\n",
+       " 'pos_image']"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "extrafeats.all_1D_features"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "These features are described as functions in extrafeats.py. You can find the docstrings of these functions by using the help function in Python. You can find more documentation of the features here: https://efel.readthedocs.io/en/latest/"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Help on function peak_to_valley in module efel.pyfeatures.extrafeats:\n",
+      "\n",
+      "peak_to_valley(waveforms, sampling_frequency)\n",
+      "    Time between trough and peak. If the peak precedes the trough,\n",
+      "    peak_to_valley is negative.\n",
+      "    \n",
+      "    Parameters\n",
+      "    ----------\n",
+      "    waveforms  : numpy.ndarray (num_waveforms x num_samples)\n",
+      "        waveforms to compute feature for\n",
+      "    sampling_frequency  : float\n",
+      "        rate at which the waveforms are sampled (Hz)\n",
+      "    \n",
+      "    Returns\n",
+      "    -------\n",
+      "    np.ndarray (num_waveforms)\n",
+      "        peak_to_valley in seconds\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "help(extrafeats.peak_to_valley)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's calculate get test MEA data and calculate the extracellular features."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "waveforms_fpath = \"../../tests/testdata/extrafeats/mean_waveforms.dat\"\n",
+    "\n",
+    "waveforms = np.loadtxt(waveforms_fpath)\n",
+    "waveform = np.array([waveforms[0]])\n",
+    "sampling_freq = 10000"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "array([0.0013])"
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "extrafeats.peak_to_valley(waveforms=waveform, sampling_frequency=sampling_freq)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `waveform` has to be form as specified in eFEL/tests/testdata/extrafeats/mean_waveforms.dat. We will use the `calculate_features` function from extrafeats to calculate the all the features."
```

Comment:
Maybe something like
The `waveform` has to have a structure similar to the one in eFEL/tests/testdata/extrafeats/mean_waveforms.dat